Vagrantfile

```diff
@@ -25,7 +25,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Create a private network, which allows host-only access to the machine
   # using a specific IP.
   config.vm.network "private_network", ip: "192.168.33.200"
-
+  config.vm.network :forwarded_port, guest: 8000, host: 8000
```

Comment:
maybe pick something more unique, this will compete with devstack at least, and probably others